### PR TITLE
Fix/time zone handling

### PR DESCRIPTION
**Pull request by tsteinruecken**
Handling timezones was still somewhat broken in ViUR3 (due to datastore now returning/expecting tz-aware date objects).
This removed a lot of now obsolete code and desolves the ExtendedDateTimeClass

### CODE REVIEW

File: bones/dateBone.py [hunk 1]
```diff
@@ -19,35 +19,9 @@
...
-		if "%a" in format:
-			format = format.replace("%a", translate("const_day_%s_short" % int(super(ExtendedDateTime, self).strftime("%w"))))
-		if "%A" in format:
-			format = format.replace("%A", translate("const_day_%s_long" % int(super(ExtendedDateTime, self).strftime("%w"))))
-		if "%b" in format:
-			format = format.replace("%b", translate("const_month_%s_short" % int(super(ExtendedDateTime, self).strftime("%m"))))
-		if "%B" in format:
-			format = format.replace("%B", translate("const_month_%s_long" % int(super(ExtendedDateTime, self).strftime("%m"))))
-		return super(ExtendedDateTime, self).strftime(format)
+def datetimeToTimestampp(datetimeObj: datetime) -> int:
```
Comment (sveneberth) [inline bones/dateBone.py:22]: Typo:
```suggestion
def datetimeToTimestamp(datetimeObj: datetime) -> int:
```

**Comment by tsteinruecken** [comment]
It's not used inside the server. However one project i've checked uses this so i included this function so there is no loss in functionality. Maybe we could add a deprecation warning to this function and remove it once noone depends on it? Or just drop it now and the projects have to ship one themselves?

**Comment by tsteinruecken** [comment]
@phorward Any preferences about datetimeToTimestamp? Keep or remove?

**Comment by phorward** [comment]
> @phorward Any preferences about datetimeToTimestamp? Keep or remove?

Keeping is better.

**Pull request merged**